Automattic/wp-calypso
# Refactor G Suite Purchase CTA view

#### Changes proposed in this Pull Request

* Refactor the G Suite marketing copy
* Add tests
* Create helper function to get eligible site

#### Testing instructions

Non-primary site with one custom domain no G Suite:

* Goto Domains
* Goto Email tab
* Observe G Suite Marketing copy
* Follow `What to test` from below

Non-primary site with one custom domain no G Suite:

* Goto Domains
* Click into custom domain
* Click email menu item
* Observe G Suite Marketing copy
* Follow `What to test` from below

Site with two custom domains and non-primary site with no G Suite:

* Goto Domains
* Click into non-primary custom domain
* Click email menu item
* Observe G Suite Marketing copy
* Follow `What to test` from below

What to test:

* Observe domain in header
* Observe domain in G mail copy
* Observe currency and amounts are correct
* Click add G Suite, does it go to the next screen?
* Click the learn more link, does it go to: https://en.support.wordpress.com/adding-g-suite-to-your-site/
* Does it fire the tracking event? `calypso:analytics:tracks Record event "calypso_domain_management_email_learn_more_click"`
